overwrites/w3c.json

```diff
@@ -8,9 +8,15 @@
     "WebCryptoAPI": {
         "edDraft": { "replaceWith": "https://w3c.github.io/webcrypto/Overview.html" }
     },
+    "encrypted-media": {
+        "edDraft": { "replaceWith": "https://w3c.github.io/encrypted-media/" }
+    },
     "service-workers-1": {
         "edDraft": { "replaceWith": "https://w3c.github.io/ServiceWorker/" }
     },
+    "touch-events": {
+        "edDraft": { "replaceWith": "https://w3c.github.io/uievents/" }
```

Comment:
You mean: https://w3c.github.io/touch-events/, right?

Reply:
Oops, of course!